src/main/scala/za/co/absa/statusboard/api/http/DecodeFailureHandlerImpl.scala

```diff
@@ -31,16 +32,21 @@ import za.co.absa.statusboard.model.ErrorResponse.BadRequestResponse
 import zio.Task
 
 object DecodeFailureHandlerImpl extends DecodeFailureHandler[Task] {
+  private val logger = LoggerFactory.getLogger(getClass)
+
   override def apply(ctx: DecodeFailureContext)(implicit monad: MonadError[Task]): Task[Option[ValuedEndpointOutput[_]]] = {
     monad.unit(
       respond(ctx).map { case (sc, hs) =>
         val message = if (sc == StatusCode.Unauthorized) "Unauthorized" else ctx.failure match {
           case DecodeResult.Missing => s"Decoding error - missing value."
           case DecodeResult.Multiple(vs) => s"Decoding error - $vs."
-          case DecodeResult.Error(original, _) => s"Decoding error for an input value '$original'."
+          case DecodeResult.Error(original, message) => s"Decoding error '$message' for an input value '$original'."
```

Comment:
To avoid name confusion:
```suggestion
          case DecodeResult.Error(original, errorMessage) => s"Decoding error '$errorMessage' for an input value '$original'."
```